data/mil2525d/utilities/military-features-utilities/Bc2_and_C_CSV_generation_for_domains/Script/GenerateDomainCSV.py

```diff
@@ -0,0 +1,116 @@
+# ----------------------------------------------------------------------------------
+# Copyright 2016 Esri
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+# GenerateDomainCSVs.py
+# Description: Reads the All_ID_Mapping_Original CSV file and
+# selects rows with Legacy Key values of 10 characters, generates
+# CSV file for the generation of 2525C or 2525Bc2 geodatabase domains.
+# Requirements: ArcGIS Desktop
+# ----------------------------------------------------------------------------------
+
+# Import arcpy module
+import arcpy, os, sys, traceback
+import csv as csv
+
+
+# variables:
+filter_codes_table_Copy = "in_memory\\filter_codes_table_Copy"
+All_ID_Mapping_Original_Copy = "in_memory\\All_ID_Mapping_Original_Copy"
+All_ID_Mapping_Sel_10_char = "in_memory\\All_ID_Mapping_Sel_10_char"
+CharlieName = ""
+
+### FOR TESTING ###choose_standard = "C"   # permissible values are B2 or C
```

Comment:
Another way to do this is - in `__main__` below if the arcpy.GetParameter is None or '' set it to these default/test values and they will use these when run outside of Desktop (in the IDE/debugger)